test/moonwall.config.json

```diff
@@ -513,7 +513,7 @@
             "name": "dev_flashbox",
             "envVars": ["DEBUG_COLORS=1"],
             "testFileDir": ["suites/common-all", "suites/common-tanssi"],
-            "runScripts": ["compile-wasm.ts compile -b ../target/release/tanssi-node -o wasm -c flashbox_dev"],
+            "runScripts": ["compile-wasm.ts compile -b ../target/release/tanssi-node -o wasm -c flashbox_dev -a \"--mock-container-chain=2000 --mock-container-chain=2001\""],
```

Comment:
Is this needed? You do `.unwrap_or(vec![2000, 2001])` if the arg is `None`, so it should be the same without this new arg. (I didn't test it)

Reply:
Still pretty convinced that you don't need this, because this command is used to compile the runtime wasm, and that does not depend on the genesis state, only on the genesis runtime wasm.

Reply:
I added it so it replicates the previous behavior of having 2000 and 2001 as mock containers. I now do `.unwrap_or(vec![])` in the `load_spec` fn, to allow generating specs without any mocks.
`compile-wasm.ts` does indeed compile the runtime wasm, but it's also where the [test runtime spec is generated](https://github.com/moondance-labs/tanssi/blob/fg/fix-flashbox-spec/test/scripts/compile-wasm.ts#L97) and saved as `tmp/dancebox_dev-raw.json`, which is the one that's later [used by the TS tests](https://github.com/moondance-labs/tanssi/blob/fg/fix-flashbox-spec/test/moonwall.config.json#L29)